simplemind/models.py

```
@@ -1,10 +1,12 @@
 from pydantic import BaseModel
-from typing import Any, ClassVar
+from typing import Any, Dict, List, Optional
+import uuid
```

Ruff

`uuid` imported but unused

Remove unused import: `uuid`

(F401)